packages/venia-ui/lib/components/Link/link.js

```diff
@@ -3,38 +3,40 @@ import { Link as RouterLink } from 'react-router-dom';
 import { bool } from 'prop-types';
 import useLink from '@magento/peregrine/lib/talons/Link/useLink';
 
-const removeProps = (props, exclude) => {
-    return Object.fromEntries(
-        Object.entries(props).filter(([name]) => !exclude.includes(name))
-    );
-};
-
 /**
  * @kind functional component
  *
- * @property {bool} prefetchType `true` activates prefetching the target page when the link becomes visible in the viewport.
+ * @property {bool} shouldPrefetch `true` activates prefetching the target page when the link becomes visible in the viewport.
  * @property {string} to From [react-router-dom Link](https://knowbody.github.io/react-router-docs/api/Link.html). The absolute path to the target page of the link. Uses the `to` prop from the `react-router-dom Link`.
  *
  * @example <caption>Basic usage</caption>
- * <Link prefetchType={true} to="/about/">About Us</Link>
+ * <Link shouldPrefetch={true} to="/about/">About Us</Link>
  */
 const Link = props => {
-    const talonProps = useLink(props);
+    // TODO: remove `prefetchType`
```

Comment:
Should this TODO be removed?

Reply:
No, the prop still exists so as not to be a breaking change. We can actually remove it in a future major release, and delete the comment at that time.